### PR TITLE
college data coming from db on List page and prettier format

### PR DESCRIPTION
**Pull request by valamuri2020**

### CODE REVIEW

**Comment by valamuri2020** [comment]
please hold up on merging this for now. I found that the logged in user is not connected directly to their list data. Once I fix this, I'll let you know to go ahead with the merge.

**Comment by valamuri2020** [comment]
logged in user is now connected to db. you can go ahead now!

**Review by Kshitij-Kumar123 — APPROVED** [review]
changes made to the codebase work as expected.  Good job!

**Pull request merged**
